### PR TITLE
User service integrate with API

### LINKED ISSUE #23: Integrate user service with API

### PR DESCRIPTION
**Pull request by Duveshka**

### CODE REVIEW

File: ArcadiaParties-SPA/src/app/user/services/user.service.ts
Comment (laska-sl) [inline ArcadiaParties-SPA/src/app/user/services/user.service.ts]: Internal after external

File: ArcadiaParties-SPA/src/app/user/effects/effect.ts
Comment (laska-sl) [inline ArcadiaParties-SPA/src/app/user/effects/effect.ts:10]: 7-8 empty line. Can't highlight, cause they were here before this PR

File: ArcadiaParties-SPA/src/app/user/effects/effect.ts
Comment (sponkrashin) [inline ArcadiaParties-SPA/src/app/user/effects/effect.ts]: You don't need to use pipe again.

File: ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs
Comment (sponkrashin) [inline ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs]: User of departments aren't related to calendar, they are still just users of the department. It doesn't make sense.

**Review by sponkrashin — CHANGES_REQUESTED** [review]

File: ArcadiaParties-SPA/src/app/user/effects/effect.ts
Comment (Duveshka) [reply ArcadiaParties-SPA/src/app/user/effects/effect.ts:10]: It deleted

File: ArcadiaParties-SPA/src/app/user/services/user.service.ts
Comment (Duveshka) [reply ArcadiaParties-SPA/src/app/user/services/user.service.ts]: Fixed

File: ArcadiaParties-SPA/src/app/user/effects/effect.ts
Comment (Duveshka) [reply ArcadiaParties-SPA/src/app/user/effects/effect.ts]: Fixed

File: ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs
Comment (Duveshka) [reply ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs]: Renamed

File: ArcadiaParties.Data.Abstractions/DTOs/UsersOfDepartmentDTO.cs
Comment (sponkrashin) [inline ArcadiaParties.Data.Abstractions/DTOs/UsersOfDepartmentDTO.cs]: Revert it back.

File: ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs
Comment (sponkrashin) [inline ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs]: Revert it back.

File: ArcadiaParties.API/Controllers/CalendarController.cs
Comment (sponkrashin) [inline ArcadiaParties.API/Controllers/CalendarController.cs]: Revert it back.

File: ArcadiaParties.Data/Repositories/UserRepository.cs
Comment (sponkrashin) [inline ArcadiaParties.Data/Repositories/UserRepository.cs]: Revert it back.

**Review by sponkrashin — CHANGES_REQUESTED** [review]

File: ArcadiaParties.Data/Repositories/UserRepository.cs
Comment (Duveshka) [reply ArcadiaParties.Data/Repositories/UserRepository.cs]: Reverted

File: ArcadiaParties.API/Controllers/CalendarController.cs
Comment (Duveshka) [reply ArcadiaParties.API/Controllers/CalendarController.cs]: Reverted

File: ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs
Comment (Duveshka) [reply ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs]: Reverted

File: ArcadiaParties.Data.Abstractions/DTOs/UsersOfDepartmentDTO.cs
Comment (Duveshka) [reply ArcadiaParties.Data.Abstractions/DTOs/UsersOfDepartmentDTO.cs]: Reverted

File: ArcadiaParties.Data.Abstractions/DTOs/UsersForCalendarDTO.cs
Comment (sponkrashin) [inline ArcadiaParties.Data.Abstractions/DTOs/UsersForCalendarDTO.cs]: Revert naming.

**Review by sponkrashin — CHANGES_REQUESTED** [review]

File: ArcadiaParties.Data.Abstractions/DTOs/UsersForCalendarDTO.cs
Comment (Duveshka) [reply ArcadiaParties.Data.Abstractions/DTOs/UsersForCalendarDTO.cs]: Reverted

**Review by sponkrashin — DISMISSED** [review]
Wait until base PR is merged. Then merge this PR to develop.

**Review by sponkrashin — APPROVED** [review]

**Pull request merged**
